File: data/readable_format/fortran/file1939.f90
program file_existence_check
    implicit none
    logical :: file_exists

    inquire(file='test_file.txt', exist=file_exists)
    if (file_exists) then
        print *, 'File exists.'
    else
        print *, 'File does not exist.'
    end if
end program file_existence_check

File: data/readable_format/cpp/file1939.cpp
#include <iostream>
#include <fstream>

int main() {
    std::ifstream file("test_file.txt");

    if (file.good()) {
        std::cout << "File exists." << std::endl;
    } else {
        std::cout << "File does not exist." << std::endl;
    }

    return 0;
}